<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaySpeed</class>
 <widget class="QWidget" name="PlaySpeed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>80</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#PlaySpeed{
	background-color: transparent;
	border: none;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="PlaySpeedBg" native="true">
     <property name="styleSheet">
      <string notr="true">#PlaySpeedBg
{
	background-color: #3D3B3B;
	border-radius: 10px;
}

QPushButton{
	border: none;
	border-radius: 4px;
	color: #FFFFFF;
	font-size: 12px;
	font-family: '微软雅黑';
}


QPushButton:hover{
	background-color: rgba(255,255,255,0.41);

}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <item>
       <widget class="QPushButton" name="speed20">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>X2.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="speed15">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>X1.5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="speed10">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>X1.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="speed05">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>X0.5</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
